utils/handler.ts

```diff
@@ -73,3 +73,42 @@ export const handleCopy = async (value: string) => {
 export const handleNftImageLoadError = (e: React.SyntheticEvent<HTMLImageElement, Event>) => {
   e.currentTarget.src = '/images/nft-placeholder.svg'
 }
+
+// return a sorter array
+export const handleSorterArrayAboutPath = (url, sorters, sorterValueEnum = null) => {
+  const params = url.slice(url.indexOf('?') + 1)
+  const sorterParamsArray = []
+
+  const searchParams = new URLSearchParams(params)
```

Comment:
```suggestion
  const searchParams = new URL(url).searchParams
```

Reply:
The `aspath` from the `next/router` seems invalid when create a URL object.